app/assets/javascripts/admin/addon/services/admin-search-data-source.js

```diff
@@ -0,0 +1,317 @@
+import { tracked } from "@glimmer/tracking";
+import Service, { service } from "@ember/service";
+import { adminRouteValid } from "discourse/lib/admin-utilities";
+import { ajax } from "discourse/lib/ajax";
+import escapeRegExp from "discourse/lib/escape-regexp";
+import getURL from "discourse/lib/get-url";
+import PreloadStore from "discourse/lib/preload-store";
+import { ADMIN_NAV_MAP } from "discourse/lib/sidebar/admin-nav-map";
+import I18n, { i18n } from "discourse-i18n";
+
+// TODO (martin) Move this to javascript.rake constants, use on server too
+export const RESULT_TYPES = ["page", "setting", "theme", "component", "report"];
+const SEPARATOR = ">";
```

Comment:
Would be cool if we can use the `d-icon-angle-right` FA that we use for the breadcrumbs.

Reply:
Yes might do that in the next PR, the separator is nice and easy for now